src/app/core/store/shopping/product-listing/product-listing.effects.ts

```diff
@@ -102,7 +102,8 @@ export class ProductListingEffects {
               page: p > 1 ? p : undefined, // same content for 0, 1 & undefined
               filters,
             };
-          })
+          }),
+          take(1)
```

Comment:
take(1) at this point is blocking the dispatch of `loadMoreProductsForParams` when the user is navigating away from a PLP. Preventing clean up operations on the store (e.g. shopping/productListing/currentSettings/xyz/filters) to take place.